spec/converse.js

```diff
@@ -369,7 +369,7 @@ describe("Converse", function() {
                         'emoji_categories': { 'travel': ':motorcycle:', 'food': ':burger:' },
                     });
 
-                    expect(_converse.api.settings.get('emoji_categories')?.travel).toBe(':rocket:');
+                    expect(_converse.api.settings.get('emoji_categories').travel).toBe(':rocket:');
                     expect(_converse.api.settings.get('emoji_categories')?.food).toBe(undefined);
```

Comment:
Hey, not that it's super important, but there is one more places where optional chaining is used and not necessary.

Reply:
Yup, no worries :-) Thank you!